### PR TITLE
feat: add spec for `pmset`

### LINKED ISSUE #1181: [pmset] missing spec
### Sanity checks

- [X] My issue relates to a specific CLI completion spec (e.g. `git checkout` is missing options in `git` completion spec). If your issue is more general, please create your issue here: [withfig/fig](https://github.com/withfig/fig/issues/new?assignees=&labels=NEED_TO_LABEL&template=main_issue_template.yml)
- [X] I have searched [github.com/withfig/autocomplete/issues](https://github.com/withfig/autocomplete/issues?q=) and there are no duplicates of my issue
- [X] I would like to work on this.

### What CLI tool does this relate to?

pmset

### Which statement makes the most sense?

This completion spec does not exist

### Issue Details

Man page: https://ss64.com/osx/pmset.html

### PR DESCRIPTION
**Pull request by bolshoytoster**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Adds spec
**What is the current behavior? (You can also link to an open issue here)**
The spec doesn't exist
**What is the new behavior (if this is a feature change)?**
The spec exists
**Additional info:**
Closes #1181

### CODE REVIEW

File: src/pmset.ts [hunk 1]
```diff
@@ -0,0 +1,287 @@
+const settings: Fig.suggestion[] = [
```
Comment (fedeci) [inline src/pmset.ts:1]:
```suggestion
const settings: Fig.Suggestion[] = [
```

File: src/pmset.ts [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+      "This setting affects how macOS networking presents shared network services during system sleep",
+  },
+  {
+    name: "destroyfvkeyonstandby",
+    description:
+      "Destroy File Vault Key when going to standby mode. (value: 1 - Destroy, 0 - Retain)",
+  },
+];
+
+const events: Fig.suggestion[] = [
```
Comment (fedeci) [inline src/pmset.ts:90]:
```suggestion
const events: string[] = [
```

File: src/pmset.ts [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+          name: "UUID",
+          description: "Used for historydetailed",
+          isOptional: true,
+        },
+      ],
+    },
+    {
+      name: "schedule",
+      description: "For setting up one-time power events",
+      options: { name: "cancel" },
```
Comment (fedeci) [inline src/pmset.ts:230]:
```suggestion
      options: [{ name: "cancel" }],
```

File: src/pmset.ts [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+          name: "date/time",
+          description:
+            '"MM/dd/yy HH:mm:ss" (in 24 hour format; must be in quotes)',
+        },
+      ],
+    },
+    {
+      name: "repeat",
+      description: "For setting up daily/weekly power on and power off events",
+      subcommands: { name: "cancel" },
```
Comment (fedeci) [inline src/pmset.ts:243]:
```suggestion
      subcommands: [{ name: "cancel" }],
```

**Review by fedeci — CHANGES_REQUESTED** [review]

File: src/pmset.ts (hunk 2 above)
Comment (bolshoytoster) [reply src/pmset.ts]: @fedeci is there a reason `string[]` is preferred, since it's used as a suggestion `Fig.Suggestion[]` is more clear.

Comment (bolshoytoster) [reply src/pmset.ts]: @fedeci fair enough

**Pull request merged**